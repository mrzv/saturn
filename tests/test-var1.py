import time

#---#

print("Evaluating")
a = 5
time.sleep(5)
print("Got a=",a)

#var> a

print(a)

#chk>

print(a + 5)
