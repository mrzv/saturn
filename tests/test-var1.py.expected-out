>>> import time                                                                 


>>> print("Evaluating")                                                         
... a = 5                                                                       
... time.sleep(5)                                                               
... print("Got a=",a)                                                           

Evaluating
Got a= 5

>>> print(a)                                                                    

5

>>> print(a + 5)                                                                

10

