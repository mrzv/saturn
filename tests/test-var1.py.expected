import time

#---#

print("Evaluating")
a = 5
time.sleep(5)
print("Got a=",a)

#var> a
#var>{{{
#var>gANDIAAsgJFS4sAE025TaR2frmmDHCiXGlX6OHv5JlLaUAWRcQAugANLBS4=
#var>}}}
#o> Evaluating
#o> Got a= 5

print(a)

#o> 5
#chk>{{{
#chk>gANDIJKLas9byFEdZYuqEwXzwM30bIY8j9+SEZnfdw8vBUDEcQAugAN9cQAoWAwAAABfX2J1aWx0aW5z
#chk>X19xAWNidWlsdGlucwpfX2RpY3RfXwpYBAAAAHRpbWVxAmNkaWxsLl9kaWxsCl9pbXBvcnRfbW9kdWxl
#chk>CnEDWAQAAAB0aW1lcQSFcQVScQZYAQAAAGFxB0sFdS4=
#chk>}}}

print(a + 5)
#o> 10
